src/navigation/Tab/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ReactElement } from 'react';
+import TabMui, { TabProps } from '@material-ui/core/Tab';
+import TabsMui, { TabsProps } from '@material-ui/core/Tabs';
+import { withStyles } from '@material-ui/core/styles';
+import styled from 'styled-components';
+
+import theme from '../../theme';
+import { IconType } from '../../dataDisplay/Icon';
+import IconText from '../../dataDisplay/IconText';
+import Text from '../../dataDisplay/Text';
+
+export type Item = {
+  id: string;
+  icon?: keyof IconType;
+  label: string;
+  customContent?: ReactElement;
+  disabled?: boolean;
+};
+
+export type Props = {
+  onChange: (selectedIndex: string) => void;
+  items: Array<Item>;
+  selectedTab: string;
+  variant?: 'outlined' | 'contained';
+  fullWidth?: Boolean;
+};
+
+const TabWrapper = styled.div<{ variantStyle: string }>`
+  border-bottom: ${({ variantStyle }) =>
+    variantStyle === 'outlined'
+      ? '1px solid ' + theme.colors.overlay.color
```

Comment:
I think theme gets passed as props.theme, doesn't it?

Reply:
Probably I was wrong about that. So is our goal to make the components standalone? E.g props.theme is not available (ThemeProvider is not used). If yes, your usage was correct